datasets/austin_crime/_images/run_csv_transform_kub/csv_transform.py

```diff
@@ -0,0 +1,252 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import datetime
+import glob
+import json
+import logging
+import math
+import os
+import pathlib
+import re
+import subprocess
+import typing
+
+import pandas as pd
+from google.cloud import storage
+
+
+def main(
+    source_url: typing.List[str],
+    source_file: typing.List[pathlib.Path],
+    source_files_path: str,
+    target_file: pathlib.Path,
+    target_gcs_bucket: str,
+    target_gcs_path: str,
+    headers: typing.List[str],
+    rename_mappings: dict,
+) -> None:
+
+    logging.info(
+        "Austin crime process started at "
+        + str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+    )
+
+    logging.info("creating 'files' folder")
+    pathlib.Path("./files").mkdir(parents=True, exist_ok=True)
+
+    logging.info("Downloading file ...")
+    download_file(source_url, source_file)
+
+    logging.info("Opening files...")
+    df = read_files(source_files_path)
+
+    logging.info("Transform: Rename columns...")
+    rename_headers(df, rename_mappings)
+
+    logging.info("Transform: Cleaning up column location_description...")
+    df["location_description"] = (
+        df["location_description"]
+        .astype("|S")
+        .str.decode("utf-8")
+        .apply(reg_exp_tranformation, args=(r"\s{2,}", ""))
+    )
```

Comment:
Perhaps add a comment on what kind of "cleanup" is happening here.

Or even better, extract this chunk of code into a function, and name the function in a descriptive way
```
def remove_two_or_more_consecutive_whitespaces(...):
  # ...
```

Reply:
I added comments